jslint4java-maven-plugin/src/main/java/com/googlecode/jslint4java/maven/JSLintMojo.java

```diff
@@ -263,7 +263,7 @@ private JSLintResult lintFile(JSLint jsLint, File file) throws MojoExecutionExce
         } catch (IOException e) {
             throw new MojoExecutionException("problem whilst linting " + file, e);
         } finally {
-            Closeables.closeQuietly(reader);
```

Comment:
Why the switch to apache commons? I don't mind, I'm just curious.

Reply:
The guava version was removed: https://github.com/google/guava/issues/1118